app/models/mixins/compliance_mixin.rb

```diff
@@ -9,7 +9,9 @@ module ComplianceMixin
              :inverse_of => :resource,
              :class_name => "Compliance"
 
-    supports :check_compliance
+    supports :check_compliance do
+      unsupported_reason_add(:check_compliance, _("No compliance policies assigned")) unless has_compliance_policies?
```

Comment:
Comment on i18n that might have gotten lost in the other outdated comment stream: https://github.com/ManageIQ/manageiq/pull/21533#discussion_r741215956

Reply:
I went ahead and rolled this back per your second comment.

We probably want to handle the i18n when we are constructing the error messages later on.